packages/wonder-blocks-tooltip/components/tooltip-anchor.js

```diff
@@ -0,0 +1,168 @@
+// @flow
+/**
+ * This component turns the given content into an accessible anchor for
+ * positioning and displaying tooltips.
+ */
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+
+import TooltipPortalMounter from "./tooltip-portal-mounter.js";
+
+type Props = {|
+    // A method that renders the content for anchoring the tooltip.
+    // This must return a TooltipPortalMounter component.
+    children: (active: boolean) => React.Element<typeof TooltipPortalMounter>,
+
+    // Callback to be invoked when the anchored content is mounted.
+    // This provides a reference to the anchored content, which can then be
+    // used for calculating tooltip bubble positioning.
+    anchorRef: (?Element) => mixed,
+
+    // When true, the element wrapped by the anchor will be given tabindex=0
+    // to make it keyboard focusable. This value defaults to true. One might set
+    // this to false in circumstances where the wrapped component already can
+    // receive focus or contains an element that can.
```

Comment:
It's too bad there isn't an easy way to check this.

Reply:
Agreed. For now, I think this works well enough.